samples/bookinfo/kube/mixer-rule-deny-label.yaml

```diff
@@ -12,16 +11,14 @@ apiVersion: "config.istio.io/v1alpha2"
 kind: checknothing
 metadata:
   name: denyreviewsv3request
-  namespace: istio-system
 spec:
 ---
 apiVersion: "config.istio.io/v1alpha2"
 kind: rule
 metadata:
   name: denyreviewsv3
-  namespace: istio-system
 spec:
   match: destination.labels["app"] == "ratings" && source.labels["app"]=="reviews" && source.labels["version"] == "v3"
   actions:
-  - handler: denyreviewsv3handler.denier.istio-system
-    instances: [ denyreviewsv3request.checknothing.istio-system ]
+  - handler: denyreviewsv3handler.denier.default
+    instances: [ denyreviewsv3request.checknothing.default ]
```

Comment:
drive-by comment: you can omit those `.default` suffix, mixer will find out the handler and instances by filling the namespace when omitted. cc: @mandarjog